activesupport/lib/active_support/number_helper/number_to_rounded_converter.rb

```diff
@@ -46,10 +46,17 @@ def strip_insignificant_zeros
           options[:strip_insignificant_zeros]
         end
 
+        def truncate_trailing_zeros
```

Comment:
I don't love this name because it implies `50.50` should be `50.5` but the tests indicate that it's actually going to return `50.50`. It also means that `truncate_trailing_zeros` and `strip_insignificant_zeros` are somewhat competing options which is confusing for the user. Do we have any other APIs in Rails that do this we can copy?

Reply:
@eileencodes you're right and I 100% agree. I don't like the name either, I think it would feel off from a user-perspective. I'm not the greatest at math / science terminology, but the term `strip_final_zeros` feels more elegant to me based on these two links: 
1. [Chemistry StackExchange](https://chemistry.stackexchange.com/questions/74560/what-do-final-zeros-to-the-right-of-decimal-point-indicate)
2. [Significant figures explanation](https://www.ruf.rice.edu/~kekule/SignificantFigureRules1.pdf), specifically "A final zero or trailing zeros in the decimal portion ONLY are significant."

---
Options in my head currently:
- `strip_final_zeros`
- `strip_trailing_zeros`
- `strip_trailing_zeros_after_decimal_point`
- `strip_tzad`